docs/_articles/en/troubleshooting.md

```diff
@@ -37,7 +37,7 @@ This is working as designed. Some rules are default-enabled by eslint while some
 
 ### Commands display `Javascript is not currently supported by the PMD engine`.
 
-Version 6.22.0 of PMD has a [Known Issue](https://github.com/pmd/pmd/issues/2081) that causes a Java OutOfMemoryError when scanning some Javascript files. Scanning Javascript with PMD has been removed from the current version of the Salesforce CLI Scanner plug-in. We plan to restore this feature in a future version.
+Version {{ site.data.versions.pmd }} of PMD has a [Known Issue](https://github.com/pmd/pmd/issues/2081) that causes a Java OutOfMemoryError when scanning some Javascript files. Scanning Javascript with PMD has been removed from the current version of the Salesforce CLI Scanner plug-in. We plan to restore this feature in a future version.
```

Comment:
Re-reading this, in this case, should we hard code to 6.22 ?

Reply:
I should've caught this. You are right. Also, we should revisit this section when we upgrade to 6.27.0.

Reply:
ok, let me hard code for now. Just so this does not change if we upgrade and miss to edit this section.

Reply:
@ranekere-sfdc I think this should be hard coded to `6.x`, not `6.22`.